satellite/orders/endpoint.go

```diff
@@ -184,6 +184,17 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 			}
 			continue
 		}
+		if len(bucketID) == 0 {
+			endpoint.log.Warn("received serial number doesn't exist in DB", zap.String("serial number", orderLimit.SerialNumber.String()))
```

Comment:
I think `UseSerialNumber` should return an error if that happens.

Reply:
ok, I wasn't sure what to choose

Reply:
fixed